intranet/utils/session.py

```diff
@@ -0,0 +1,29 @@
+from redis_sessions.session import SessionStore as BaseSessionStore
+
+
+class SessionStore(BaseSessionStore):  # pylint: disable=abstract-method
```

Comment:
Given that this affects authentication and custom tinkering with our authentication backend, I would recommend some sort of unit tests.

Reply:
Unfortunately, I don't think that's possible, especially given that in `settings/__init__.py` we don't set `SESSION_ENGINE` during tests. We may just have to make sure to carefully explain any changes in comments.